mvc/models/model-binding.rst

```diff
@@ -1,8 +1,113 @@
-.. include:: /../common/stub-topic.txt
+Model Binding
+=============
 
-|stub-icon| Model Binding Request Data
-======================================
+By `Rachel Appel <http://github.com/rachelappel>`_
 
-.. include:: /../common/stub-notice.txt
+In this article
 
-.. _issue: https://github.com/aspnet/Docs/issues/114
+- `Introduction to model binding`_
+- `How model binding works`_
+- `Customize default model binding behavior with attributes`_
+- `Binding formatted data from the request body`_ 
+- `Resources`_
+
+Introduction to model binding
+-----------------------------
+Model binders are objects in the ASP.NET MVC framework that map data from HTTP requests to action method parameters. The parameters may be simple types such as strings, integers, or floats, or they may be complex objects representing an enterprise-sized model. This is a great feature of MVC because mapping incoming data to a counterpart is an often repeated scenario, regardless of size or complexity of the data. MVC solves this problem by abstracting binding away so developers don't have to keep rewriting a slightly different version of that same code in every app. In previous versions of ASP, including Classic ASP and WebForms, you would extract the HTTP POST or GET values manually, validate the data, and then send that data off to the database. Not much of the process was automated, so you would often see code similar the following example that captures two form values, puts them in variables and TextBox objects, storing them for their journey to the database:
```

Comment:
>  In previous versions of ASP, including Classic ASP and WebForms

I would prefer to not motivate features with comparisons to older frameworks. I would cut the entire retro sample.This kind of text is only relevant for a brief period of time and to a specific audience and then it goes stale (ex a Linux dev would read this text and not know what Class ASP or WebForms are). Instead, I think it's better to have declarative statements about what the feature is and why it is useful. Ex Model binding makes it easy to map request data in a variety of formats to strongly typed objects that are part of your application domain.